pyang/plugins/sid.py

```diff
@@ -575,6 +588,9 @@ def collect_module_items(self, module):
         if 'item' not in self.content:
             self.content['item'] = []
 
+        if 'key-mapping' not in self.content: 
```

Comment:
Does that leave an empty entry if --sid-extension is not given?

Reply:
sorry, this is not too easy for me to set up a test for

Reply:
You're right, the key-mapping was appearing even if sid-extension was not set.

I fixed it line 598,